apps/auth0/package.json

```diff
@@ -9,11 +9,13 @@
     "build:babel": "../../scripts/build-babel.sh build",
     "watch:babel": "../../scripts/build-babel.sh watch",
     "typecheck": "tsc",
-    "deploy": "a0deploy import --input_file ./tenant.yaml"
+    "deploy": "yarn sync:dependencies && yarn deploy:tennant",
```

Comment:
i guess this makes sense for dev and prod deployment since these are the only places we run this anyway, but I thought maybe we want to build in PR stage? I've added a comment to the build script to throw if the deps aren't found, if you did that it would be worth running the build pre-master merge, otherwise this is good

Reply:
If the dependencies aren't found then then they aren't in package.json and the unit tests will fail. I think the risk is much higher the other way round but you said not to overthink it